### ISSUE TITLE
bundle storm-mesos code into release tarball

### ISSUE THREAD

**Issue by erikdw** [mesos/storm#163]
At least while debugging / testing it will be invaluable to include the actual source code, since we perform many iterations.  So perhaps we should gate this on the version having `-SNAPSHOT` in it.  Since the `build-release.sh` script is responsible for the [creation of the package tarball](https://github.com/mesos/storm/blob/92f5b047230ad36e1779f983f841431a87c3e4df/bin/build-release.sh#L92-L122), that should be trivial to implement.

**Comment by erikdw** [comment]
Propose to use this method:
```
stashRef=`git stash create`
git archive --format=tar.gz -o storm-mesos-src-$stashRef.tgz --prefix storm-mesos-src-$stashRef/ $stashRef
```

That will create a dangling commit / ref.  which can be later be seen via `git fsck --unreachable`, and cleaned up via `git gc --prune=now`.

One downside with this method is that it won't include new files which haven't been added.  However, it is more future-proof against changes in the directory structure and set of files and avoids inclusion of various binaries.

**Comment by erikdw** [comment]
Fixed via #165.
